modules/oadp-usecase-include-ca-cert-backup.adoc

```diff
@@ -0,0 +1,276 @@
+----
+
+. Create the `cloud-credentials` secret with the `cloud-credentials` file content as shown in the following command:
++
+[source,terminal]
+----
+$ oc create secret generic \
+  cloud-credentials \
+  -n openshift-adp \
+  --from-file cloud=cloud-credentials
+----
+
+. Extract the service CA certificate from the `openshift-service-ca.crt` config map by running the following command. Ensure that you encode the certificate in `Base64` format and `echo` the value to use in a later step.
++
+[source,terminal]
+----
+$ oc get cm/openshift-service-ca.crt \
+  -o jsonpath='{.data.service-ca\.crt}' | base64 -w0; echo
+----
++
+.Example output
++
+[source,terminal]
+----
+LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0...
+....gpwOHMwaG9CRmk5a3....FLS0tLS0K
+----
+
+. Configure the Data Protection Application (DPA) with the bucket name and CA certificate as shown in the following example:
++
+.Example DPA
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: oadp-backup
+  namespace: openshift-adp
+spec:
+  configuration:
+    nodeAgent:
+      enable: true
+      uploaderType: kopia
+    velero:
+      defaultPlugins:
+        - aws
+        - openshift
+        - csi
+      defaultSnapshotMoveData: true 
+  backupLocations:
+    - velero:
+        config:
+          profile: "default"
+          region: noobaa
+          s3Url: https://s3.openshift-storage.svc 
+          s3ForcePathStyle: "true" 
+          insecureSkipTLSVerify: "false" # <1>
+        provider: aws
+        default: true
+        credential:
+          key: cloud
+          name:  cloud-credentials 
+        objectStorage:
+          bucket: <bucket_name> # <2>
+          prefix: oadp
+          caCert: <ca_cert> # <3>
+----
+<1> The `insecureSkipTLSVerify` flag can be set to either `true` or `false`. If set to "true", SSL/TLS security is disabled. If set to `false`, SSL/TLS security is enabled.
+<2> Specify the name of the bucket extracted in an earlier step.
+<3> Copy and paste the `Base64` encoded certificate from an earlier step.
+
+. Create the DPA by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f <dpa_filename> 
+----
+
+. Verify that the DPA has been created successfully by running the following command:In the example output, you can see the `status` object has the `type` field set to `Reconciled`. This means the DPA has been successfully created.
++
+[source,terminal]
+----
+$ oc get dpa -o yaml
+----
++
+.Example output
++
+[source,yaml]
+----
+apiVersion: v1
+items:
+- apiVersion: oadp.openshift.io/v1alpha1
+  kind: DataProtectionApplication
+  metadata:
+    namespace: openshift-adp
+    #...#
+  spec:
+    backupLocations:
+    - velero:
+        config:
+          #...#
+  status:
+    conditions:
+    - lastTransitionTime: "20....9:54:02Z"
+      message: Reconcile complete
+      reason: Complete
+      status: "True"
+      type: Reconciled
+kind: List
+metadata:
+  resourceVersion: ""
```

Comment:
It has been flagged to me in reviews for other PRs to keep the top part of the CR so that the user can identify what is the type of the CR (though is clearly obvious that is an example output). So, keeping it as is. Thanks.

Reply:
No worries. The top part is required essentially in the step's YAML where we define the CR. In the example output, you can avoid long outputs to focus on the expected output part.